docs/css/extra.css

```diff
@@ -1,26 +1,25 @@
-@import url('https://fonts.googleapis.com/css?family=Rubik');
 h1, h2, h3, h4, h5, h6 {
     font-family: 'Rubik', sans-serif;
-
 }
 
 [data-md-color-scheme="testcontainers"] {
-    --md-primary-fg-color:       #291A3F;
-    --md-accent-fg-color:        #291A3F;
+    --md-primary-fg-color:       #00bac2;
+    --md-accent-fg-color:        #361E5B;
     --md-typeset-a-color:        #0C94AA;
+    --md-primary-fg-color--dark: #291A3F;
+    --md-default-fg-color--lightest: #F2F4FE;
+    --md-footer-fg-color: #361E5B;
+    --md-footer-fg-color--light: #746C8F;
+    --md-footer-fg-color--lighter: #C3BEDE;
+    --md-footer-bg-color: #F7F9FD;
+    --md-footer-bg-color--dark: #F7F9FD;
 }
 
 .card-grid {
     display: grid;
     gap: 10px;
 }
 
-.tc-version {
```

Comment:
This is class is needed, as all module pages include an HTML span element using it. Please see https://github.com/testcontainers/testcontainers-go/blob/main/docs/modules/k3s.md?plain=1#L3 as an example.